### ISSUE TITLE
No tv/movies showing

### ISSUE THREAD

**Issue by maronsen** [iBaa/PlexConnect#408]
Hi, PlexConnect works and i can access it in the wsj app but when trying out the different tabs nothing is showing up an example is that when accessing the tvshows list and selecting the all tvshows tab it says all programs are allready watched.

Your ATV model (2 or 3) and firmware version (found under the ATV settings > general > about > Apple TV software): 
ATV-3 Version 7.2(7512)

The DNS server set on the ATV (found under the ATV settings > general > network > DNS): 192.168.1.50

The local IP address of the device that PlexConnect is installed on:
192.168.1.50

The device and operating system (including version number) that PlexConnect is installed on:
A PC running windows server 2008 r2 standard

The device and operating system (including version number) that the Plex media server (PMS) is installed on:
A PC running windows server 2008 r2 standard

The Plex media server (PMS) version number you are running (do not put 'latest'):
0.9.14.5

The local IP address of the device that the Plex media server (PMS) is installed on:
192.168.1.50

The PlexConnect version number. If using Github source then a time and date of download (look at the creation date of the folder) and if you are using any non-standard setup or a fork.
0.5-dev-101215

The contents of your 'Settings.cfg' file and Plexconnect.log(this is created at first clean shutdown of PlexConnect, using CTR+c)
[PlexConnect.log.txt](https://github.com/iBaa/PlexConnect/files/75912/PlexConnect.log.txt)
[Settings.cfg.txt](https://github.com/iBaa/PlexConnect/files/75914/Settings.cfg.txt)

I am running PlexConnect Normal exept for using my own dns server and using wsj app on appletv. 

and images
![img_0921](https://cloud.githubusercontent.com/assets/7189809/11944427/96a5449a-a846-11e5-805a-232c9857bfdf.JPG)
![img_0922](https://cloud.githubusercontent.com/assets/7189809/11944429/96a9af30-a846-11e5-845d-ca1eb0e0de48.JPG)
![img_0923](https://cloud.githubusercontent.com/assets/7189809/11944428/96a8db32-a846-11e5-9168-a0b133de981e.JPG)

**Comment by iBaa** [comment]
Well, that's what your local PMS is telling for every request - OnDeck, RecentlyAdded, ...:
`00:16:05 PlexAPI: <MediaConverter friendlyName="aronsen.priv.no:32400 Servers" size="0" />`
Did you set up PMS to host TVShows/Movies locally? How does it work with another client - eg Plex/Web running on your PMS machine, or another "good" client externally?

**Comment by maronsen** [comment]
Works fine on Plex/web accessing it through custom urls and works perfectly through local ip and plex.tv so i have no idea why plexconnect is saying i have no movies and tvshows

**Comment by iBaa** [comment]
Are you signed in to myPlex? Are you using PlexHome/managed Users?

**Comment by finkdiff** [comment]
Your  setup appears to be an unusual setup, it would help significantly filling in the template requested in the how to report an issue page on the wiki and to understand what you're setup is e.g. You are using custom url's. Why do you need to use these and not just have plex.tv do the resolution for you?

**Comment by maronsen** [comment]
Signed in to myplex and are using plex/home/managed users when trying to access one of the urls from the log with library/sections/1/all/ i am getting 401 unauthorized its probably that i am not signed in. will try that first.
On the custom urls no reason other than its easy and it looks better and it works for all other devies except plexconnect right now. but will try signing in again maybe that fixes it.

**Comment by maronsen** [comment]
You Can Close the issue fixed it.

**Comment by maronsen** [comment]
I have found out that in the logs of plexconnect it is retrieving an old plex media server token. the log shows it retrieved a plex token but comparing that to the one in my web browser when showing the xml of a tv show episode is totally different. I am wondering if plex.tv is not updating the plex-token and derfor has the old one and gives that to plexconnect when it try to poke plex.tv. Are there som way to force a update in plex?

**Comment by iBaa** [comment]
>If i logout of the server i am able to get plexconnect working if i also logout of plexconnect on the atv.

When logging out both sides, PlexConnect falls back to using PlexGDM instead of the devices' database of plex.tv.

>the log shows it retrieved a plex token but comparing that to the one in my web browser when showing the xml of a tv show episode is totally different.

Tokens are AFAIK device specific - you will get different tokens depending on the X-Plex-* data shipped to plex.tv while logging in. I don't think that there is anything wrong with the token generation... more likely with PlexConnect's host name resolution in your environment. Check the discovery portion in your log: Does PlexConnect pick the correct address (in case plex.tv responds with multiple)? Is this address reachable from PlexConnect AND aTV? Does the request to PMS go through? Is it answered correctly?

**Comment by mcmaxl** [comment]
Got the same issue with Plexconnect showing my media sections but telling me "No content available"
Was this issue resolved for you?

**Comment by iBaa** [comment]
Good question. I never got an answer to my questions... based on that I guess we could close this report.
Please post your issues in the Plex forums, attach a log recorded with the PlexConnect startup, entering Trailers, up to the "no content" note.

**Comment by dnisbet** [comment]
I may be experiencing a similar issue, I will test this later tonight. 
Basically I have migrated everything to a new server but my watched/unwatched counts seem to be wrong on some things, resulting in a negative watched count on Plex/Web (which it would appear PlexConnect isn't handling as cleanly - kind of fair enough though!).
So whilst shows are appearing in On Deck and Recently Added, any view that depends on a watched count gets me nothing.
I've fixed the watch count now (hopefully), and will check when I get home whether things are appearing properly.
